prow/plugins/releasenote/releasenote.go

```diff
@@ -275,7 +275,7 @@ func handlePR(gc githubClient, log *logrus.Entry, pr *github.PullRequestEvent) e
 			}
 			if containsNoneCommand(comments) {
 				labelToAdd = labels.ReleaseNoteNone
-			} else if !prLabels.Has(labels.ReleaseNoteLabelNeeded) {
+			} else if !prLabels.Has(labels.ReleaseNoteLabelNeeded) && !pr.PullRequest.Merged {
```

Comment:
So this prevents adding a comment but not the label (which the issue is about). Maybe we should discard events on merged PRs sooner? Is there even any valid action when then the PR is merged?

Reply:
About the first one, I noticed it and was about to modify the change.
Regarding the actions, I will look into it. Not sure if possible.

Reply:
Ok, hopefully, moved this condition to a better place.